_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -23,20 +23,27 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [semantic link][], for which all the following is true:
+This rule applies to any [semantic link][] for which all the following is true:
 
 - **link text**: the [semantic link][] element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
 - **non-link line text**: the [semantic link][] element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
 - **different hue**: the [foreground colors][foreground color] of the [semantic link][] and the [foreground color][] of the **non-link line text** elements have a [different hue][], or the [background colors][background color] of the [semantic link][] and the [background color][] of the **non-link line text** elements have a [different hue][].
 
 ## Expectation 1
 
-For each test target, there exists at least one [inspection state][] that is not [focused][] and [hovered][] at the same time and at least one [visible][] [inclusive descendant][] or [ancestor][] element that is not an [ancestor][] of the **non-link line text**, such that the [visible][] [inclusive descendant][] or the [ancestor][] element has a [distinguishable style][] from each element containing **non-link line text**.
+For each test target there exists at least one element with a [distinguishable style][] from each **non-link line text** elements for which one of the following is true:
+
+- the element is a [visible][] [inclusive descendant][] of the test target; or
+- the element is [ancestor][] of the test target that is not simultaneously an [ancestor][] of the **non-link line text** elements.
 
 ## Expectation 2
 
 Either the test target does not match the [`:any-link` pseudo-class](https://drafts.csswg.org/selectors-4/#any-link-pseudo), or Expectation 1 holds for each [link history state][] with a **different hue**.
 
+## Expectation 3
+
+Expectation 1 holds irrespectively of the test target being [focused][] or [hovered][].
```

Comment:
Just to make it clear, I understand that this requires the 4 states to be valid, correct?

Reply:
Correct